### PR TITLE
preparing the scale up

### PR DESCRIPTION
**Pull request by jmvera255**
this guide replaces scaling-up.md that was moved to `archive/`

I'm concerned the guide is too verbose at this time and maybe not what we are envisioning for conveying this type of information

@lmichael107 and @ChristinaLK please provide comments, feedback, revisions, etc.

### CODE REVIEW

File: start/scaling-up/preparing-to-scale-up.md [hunk 1]
```diff
@@ -1,92 +1,142 @@
...
+data sets; if your data spans a range of sizes, submit tests using the smallest 
+and largest data sizes to examine the range of resources that these jobs may need.
+
+- Give your test jobs, and associated HTCondor `log`, `error`, `output`, 
+and `submit` files meaningful names so you know which results refer to which tests.
+
+# Optimize Job Resource Requests
+
+In the HTCondor submit file, you must explicitly request the number of 
+cpus (i.e. cores) and the amount of disk and memory that the job needs
```
Comment (ChristinaLK) [inline start/scaling-up/preparing-to-scale-up.md:61]: I might capitalize CPUs here?

File: start/scaling-up/preparing-to-scale-up.md [hunk 2]
```diff
@@ -1,92 +1,142 @@
...
+# Submit Multiple Jobs Using A Single Submit File
+
+Once you have a single test job that compeltes successfully, the next 
+step is to submit a small batch of test jobs (e.g. 5 or 10 jobs) 
+[**using a single submit file**](https://support.opensciencegrid.org/support/solutions/articles/12000073165). Use this small-scale multi-job submission test to ensure that all jobs 
+complete successfully, produce the desired output, and do not 
+conflict with each other when submitted together. Once 
+you are confident that the jobs will complete as desired, then scale up to submitting 
+the entire set of jobs.
+
```
Comment (ChristinaLK) [inline start/scaling-up/preparing-to-scale-up.md:119]: Do we want to add a note about using `condor_history` to see how much memory/disk jobs used? The history file turns over pretty quickly, but it's much easier than looking through log files. 

I think you could add (yet another) note here about checking the log files.

**Review by ChristinaLK — COMMENTED** [review]
I think this looks good. It's certainly worth putting on the website + then we can tweak it as needed.

We should add this to our "let's make a video about..." list. Each step here would make a great 40-50 second bit of a 4-5 minute video.

File: start/scaling-up/preparing-to-scale-up.md (hunk 1 above)
Comment (jmvera255) [reply start/scaling-up/preparing-to-scale-up.md]: I have capitalized CPUs in commit  b69ebd8

File: start/scaling-up/preparing-to-scale-up.md (hunk 2 above)
Comment (jmvera255) [reply start/scaling-up/preparing-to-scale-up.md:131]: i have added note about `condor_history` as well as give an example of using `grep` to pull detail from the log file in commit  b69ebd8

**Comment by jmvera255** [comment]
@ChristinaLK @lmichael107 ready for merge?

File: start/scaling-up/preparing-to-scale-up.md [hunk 3]
```diff
@@ -1,92 +1,154 @@
...
+step is to submit a small batch of test jobs (e.g. 5 or 10 jobs) 
+[**using a single submit file**](https://support.opensciencegrid.org/support/solutions/articles/12000073165). Use this small-scale 
+multi-job submission test to ensure that all jobs complete successfully, produce the 
+desired output, and do not conflict with each other when submitted together. Once 
+you are confident that the jobs will complete as desired, then scale up to submitting 
+the entire set of jobs.
+
+## Avoid Exceeding Disk Quotas
 
 Each OSG Connect user is granted 100 GB of storage in their `/home` directory and
```
Comment (ChristinaLK) [inline start/scaling-up/preparing-to-scale-up.md:134]:
```suggestion
Each OSG Connect user is granted 50 GB of storage in their `/home` directory and 

```

**Review by ChristinaLK — APPROVED** [review]

**Pull request merged**